tasks/libs/common/auth.py

```diff
@@ -22,3 +22,16 @@ def datadog_infra_token(ctx, audience, datacenter="us1.ddbuild.io"):
     )
 
     return token
+
+
+def aws_vault_env(ctx, account: str):
```

Comment:
nit:
```suggestion
def get_aws_vault_env(ctx, account: str):
```
With the current name I would expect it to be a contextmanager rather than a function returning the vars